tianshou/trainer/onpolicy.py

```diff
@@ -42,18 +43,20 @@ def onpolicy_trainer(
     :type test_collector: :class:`~tianshou.data.Collector`
     :param int max_epoch: the maximum number of epochs for training. The
         training process might be finished before reaching the ``max_epoch``.
-    :param int step_per_epoch: the number of policy network updates, so-called
-        gradient steps, per epoch.
-    :param int collect_per_step: the number of episodes the collector would
-        collect before the network update. In other words, collect some
-        episodes and do one policy network update.
+    :param int step_per_epoch: the number of environment frames collected per epoch.
     :param int repeat_per_collect: the number of repeat time for policy
         learning, for example, set it to 2 means the policy needs to learn each
         given batch data twice.
     :param episode_per_test: the number of episodes for one policy evaluation.
     :type episode_per_test: int or list of ints
     :param int batch_size: the batch size of sample data, which is going to
         feed in the policy network.
+    :param int step_per_collect: the number of episodes the collector would
+            collect before the network update. Only either one of step_per_collect
+            and episode_per_collect can be specified.
+    :param int episode_per_collect: the number of episodes the collector would
+            collect before the network update. Only either one of step_per_collect
+            and episode_per_collect can be specified.
```

Comment:
```suggestion
    :param int step_per_collect: the number of episodes the collector would
        collect before the network update. Only either one of step_per_collect
        and episode_per_collect can be specified.
    :param int episode_per_collect: the number of episodes the collector would
        collect before the network update. Only either one of step_per_collect
        and episode_per_collect can be specified.
```

Reply:
The annotations of these two parameters are the same.

Reply:
solved